client/blocks/plan-storage/test/bar.jsx

```diff
@@ -27,84 +31,107 @@ describe( 'PlanStorageBar basic tests', () => {
 	};
 
 	test( 'should not blow up and have class .plan-storage-bar', () => {
-		const bar = shallow( <PlanStorageBar { ...props } /> );
-		assert.lengthOf( bar.find( '.plan-storage__bar' ), 1 );
+		const { container } = render( <PlanStorageBar { ...props } /> );
+		expect( container.getElementsByClassName( 'plan-storage__bar' ) ).toHaveLength( 1 );
 	} );
 
 	test( 'should render ProgressBar', () => {
-		const bar = shallow( <PlanStorageBar { ...props } /> );
-		const progressBar = bar.find( 'ProgressBar' );
-		assert.lengthOf( progressBar, 1 );
-		assert.equal( progressBar.props().value, 10 );
+		render( <PlanStorageBar { ...props } /> );
+		const progressBar = screen.queryByRole( 'progressbar' );
+		expect( progressBar ).toBeDefined();
+		expect( progressBar ).toHaveAttribute( 'aria-valuenow', '10' );
 	} );
 
 	test( 'should render when storage is limited', () => {
-		let bar;
-
-		bar = shallow( <PlanStorageBar { ...props } sitePlanSlug={ PLAN_PREMIUM } /> );
-		assert.lengthOf( bar.find( '.plan-storage__bar' ), 1 );
-
-		bar = shallow( <PlanStorageBar { ...props } sitePlanSlug={ PLAN_PREMIUM_2_YEARS } /> );
-		assert.lengthOf( bar.find( '.plan-storage__bar' ), 1 );
-
-		bar = shallow( <PlanStorageBar { ...props } sitePlanSlug={ PLAN_PERSONAL } /> );
-		assert.lengthOf( bar.find( '.plan-storage__bar' ), 1 );
-
-		bar = shallow( <PlanStorageBar { ...props } sitePlanSlug={ PLAN_PERSONAL_2_YEARS } /> );
-		assert.lengthOf( bar.find( '.plan-storage__bar' ), 1 );
-
-		bar = shallow( <PlanStorageBar { ...props } sitePlanSlug={ PLAN_FREE } /> );
-		assert.lengthOf( bar.find( '.plan-storage__bar' ), 1 );
-
-		bar = shallow( <PlanStorageBar { ...props } sitePlanSlug={ PLAN_BUSINESS } /> );
-		assert.lengthOf( bar.find( '.plan-storage__bar' ), 1 );
-
-		bar = shallow( <PlanStorageBar { ...props } sitePlanSlug={ PLAN_BUSINESS_2_YEARS } /> );
-		assert.lengthOf( bar.find( '.plan-storage__bar' ), 1 );
-
-		bar = shallow( <PlanStorageBar { ...props } sitePlanSlug={ PLAN_ECOMMERCE } /> );
-		assert.lengthOf( bar.find( '.plan-storage__bar' ), 1 );
-
-		bar = shallow( <PlanStorageBar { ...props } sitePlanSlug={ PLAN_ECOMMERCE_2_YEARS } /> );
-		assert.lengthOf( bar.find( '.plan-storage__bar' ), 1 );
+		const { container: premContainer } = render( <PlanStorageBar { ...props } /> );
```

Comment:
Why do we need two renders for this test but not for the others?

Reply:
Good catch - we don't. Removed the extra render in cf0a4f3649e86ef76c77b9d24a1c0f4318daa146.